src/main/java/com/softserve/academy/spaced/repetition/domain/DeckPrice.java

```diff
@@ -0,0 +1,54 @@
+package com.softserve.academy.spaced.repetition.domain;
+
+import com.softserve.academy.spaced.repetition.controller.dto.annotations.EntityInterface;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "deck_price")
+public class DeckPrice implements EntityInterface {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "deck_price_id")
+    private Long id;
+
+    @OneToOne
+    @JoinColumn(name = "deck_id")
+    private Deck deck;
+
+    @Column(name = "deck_price")
+    private int price;
```

Comment:
@OleksandrMazur, @maxturko there is an issue here and in CourcePrice, as I recall a NullPointerException. The null value is allowed in the database and not supported here by int primitive type. You have to change it to Integer to support null values.